examples/solid/README.MD

```diff
@@ -0,0 +1,35 @@
+[![Preview](https://user-images.githubusercontent.com/91349014/236637420-19f7aa45-9e4e-4478-b279-8f06cdf4bef4.png)](https://user-images.githubusercontent.com/91349014/236637391-b0de3060-f9ec-41d1-9351-2c4eeb295294.mp4)
+
+## Env variables
+
+Check out `.env.example` for the required environment variables.
+
+This project was created using [Create JD App](https://github.com/OrJDev/create-jd-app)
```

Comment:
Can we kill these callouts? Cool w/ the credit at the bottom tho

We should standardize these readmes, cc @markflorkowski (we can also cut an issue for it maybe?)